.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,19 @@
+
+name: "Coding Standards"
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
```

Comment:
I'd suggest using two spaces everywhere. I usually add a .editorconfig file to the .github directory to achieve that. Here is an example: https://github.com/twirphp/twirp/blob/master/.github/.editorconfig